app/app/settings.py

```diff
@@ -74,12 +74,20 @@
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'django.contrib.auth.middleware.RemoteUserMiddleware',  # added for custom SSO 
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'social_django.middleware.SocialAuthExceptionMiddleware',
     'applicationinsights.django.ApplicationInsightsMiddleware',
+    
+    'custom_auth.CustomSSOMiddleware' # added for custom SSO 
 ]
 
+# added for custom SSO 
+CREATE_USER='TRUE'
```

Comment:
Use `env.bool(environment-variable-name, default-value)` ([usage example](https://github.com/sloria/environs/blob/101e113010d0436b23bb6cd72f634977174f6820/examples/django_example.py#L20)) so that the value can be set via an environment variable as per the [12-factor app](https://12factor.net/config).